assets/pod-checkpointer-operator/daemonset.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    k8s-app: pod-checkpointer
+  name: pod-checkpointer
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      k8s-app: pod-checkpointer
+  template:
+    metadata:
+      labels:
+        k8s-app: pod-checkpointer
+      annotations:
+        checkpointer.alpha.coreos.com/checkpoint: "true"
```

Comment:
does the checkpointer need to be checkpointed? my brain hurts thinking about the implications...

Reply:
yep. the checkpointer gets checkpointed as well. It's so the static manifest exists on a reboot, and the kubelet can restart the checkpointer.